resources/js/humhub.wiki.Form.js

```diff
@@ -111,26 +134,261 @@ humhub.module('wiki.Form', function(module, require, $) {
+            client.get(url).then(function(response) {
+                if(response.success&&response.conflictingEditing) {
+                    var options = {
+                        'header': response.header,
+                        'body': response.body,
+                        'confirmText': response.confirmText,
+                        'cancelText' : response.cancelText,
+                    };
+
+                    modal.confirm(options).then(function ($confirmed) {
+                        if ($confirmed) {
+                            client.redirect(response.url);
+                        }
+                    });
+                }
+            }).catch(function(e) {
+                // module.log.error(e, true);
+            })
+        }
+    }
+
+    function insertContentIntoEditor(editorWidget, content) {
+        var view = editorWidget.editor.view;
+    
+        if (view) {
+            var parser = richtext.api.model.DOMParser.fromSchema(view.state.schema);
+            var html = content;
+            var tempDoc = new DOMParser().parseFromString(html, 'text/html');
+            var doc = parser.parse(tempDoc);
+    
+            var endPos = view.state.doc.content.size;
+    
+            var transaction = view.state.tr.insert(endPos, doc.content);
+            view.dispatch(transaction);
+        } else {
+            console.warn('ProseMirror view not found');
+        }
+    }
+
+    function requireTemplate(editorWidget) {
+        if(document.querySelector('.ProseMirror .placeholder')) {
+            $('#templateSelectModal').modal('show');
+        }
+        else {
+            $('#templateSelectModal .modal-body').empty();
+        }
+        $('#useTemplateBtn').on('click', function (e) {
+            e.preventDefault();
+            const $selected = $('#templateSelectDropdown option:selected');
+            const fetchUrl = $selected.data('url');
+        
+            if (!fetchUrl) {
+                alert('Please select a template');
+                return;
+            }
+        
+            $.get(fetchUrl, function (response) {
+                if (response.success) {
+                    const content = response.content;
+                    const titleTemplate = response.title;
+                    const placeholders = response.placeholders? JSON.parse(response.placeholders): [];
+                    const is_appendable = response.is_appendable;
+                    const appendable_content = response.appendable_content;
+            
+                    if (placeholders.length > 0) {
+                        let formHtml = '<form id="templatePlaceholderForm">';
+                        placeholders.forEach(ph => {
+                            formHtml += `
+                                <div class="form-group mb-2">
+                                    <label>${ph.description + '(' + ph.key + ')'}</label>
```

Comment:
There should be a blank in between:
```suggestion
                                    <label>${ph.description + ' (' + ph.key + ')'}</label>
```